src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml

```diff
@@ -26,6 +26,12 @@
             <tag name="serializer.normalizer" priority="-1000" />
         </service>
 
+        <!-- Json Normalizer -->
+        <service id="serializer.normalizer.json" class="Symfony\Component\Serializer\Normalizer\JsonSerializableNormalizer" public="false">
```

Comment:
Could add conflict, i think thats a good idea, means we don't need any extra code for legacy reasons.
I notice that `symfony/serializer` isn't in the `require-dev` section either which it probably should be.